connector/ldap/ldap.go

```diff
@@ -225,6 +227,14 @@ func (c *Config) openConnector(logger logrus.FieldLogger) (*ldapConnector, error
 		}
 		tlsConfig.RootCAs = rootCAs
 	}
+
+	if c.ClientKey != "" && c.ClientCert != "" {
+		cert, err := tls.LoadX509KeyPair(c.ClientCert, c.ClientKey)
+		if err != nil {
+			return nil, fmt.Errorf("ldap: load self signed certs file failed.error:%s", err)
```

Comment:
[nit] Can we please make that 
```go
fmt.Errorf("ldap: load client cert failed: %v", err)
```
just to be in line with the updated comment (thanks, btw), and the other error printouts above?